src/pathfora.js

```diff
@@ -137,6 +137,60 @@
     MODAL_CLOSE: 'modalClosed'
   };
 
+  // NOTE AB testing types
+  /**
+   * @function createABTestingModePreset
+   * @description Create an A/B testing object preset from groups list
+   * @returns {object} A/B testing object instance
+   */
+  var createABTestingModePreset = function () {
+    var groups = [];
+    var groupsSum;
+    var groupsSumRatio;
+    var i;
+    var j;
+
+    j = arguments.length;
+    for (i = 0; i < j; i++) {
+      groups.push(arguments[i]);
+    }
+
+    groupsSum = groups.reduce(function (sum, element) {
+      return sum + element;
+    });
+
+    // NOTE If groups collapse into a number greater than 1, normalize
+    if (groupsSum > 1) {
+      groupsSumRatio = 1 / groupsSum;
+
+      groups = groups.map(function (element) {
+        return element * groupsSumRatio;
+      });
+    }
+
+    return {
+      groups: groups,
+      groupsNumber: groups.length
+    };
+  };
+
+  /*
+   * @global
+   * @property abHashMD5
+   * @description Hash used as the AB testing cookie (MD5 'ab')
+   */
+  var abHashMD5 = '187ef4436122d1cc2f40dc2b92f0eba0';
```

Comment:
Nothing in particular, just a cosmetic addition, instead of putting `test_1` etc. in the cookies.